CSSS/Kernel.cs

```diff
@@ -157,6 +157,43 @@ public bool PerformTasks()
                 // so return 'true' from this function to let the
                 // calling main run loop know it can exit
                 shouldExit = true;
+
+                // If we should shutdown
+                if (config.CSSSProgramMode.HasFlag(Config.CSSSModes.Shutdown))
+                {
+                    // Run the shutdown command
+                    var process = new System.Diagnostics.Process();
+                    process.StartInfo.FileName = "shutdown";
+
+                    Console.WriteLine("Shutting down computer in 1 minute for image capture");
+
+                    logger.Info("Attempting to shut down the computer in 1 minute");

```

Comment:
Do we need both the `Console.WriteLine` and `logger.Info`?

Reply:
You requested that it output the Console.WriteLine as well as the logger. I'll remove it as there seems to be no other output produced.